cranelift/filetests/filetests/alias/simple-alias.clif

```diff
@@ -0,0 +1,92 @@
+test compile precise-output
+set opt_level=speed
+target aarch64
+
+function %f0(i64 vmctx, i32) -> i32, i32, i32, i32 {
+    gv0 = vmctx
+    gv1 = load.i64 notrap readonly aligned gv0+8
+    heap0 = static gv1, bound 0x1_0000_0000, offset_guard 0x8000_0000, index_type i32
+    fn0 = %g(i64 vmctx)
+
+block0(v0: i64, v1: i32):
+    v2 = heap_addr.i64 heap0, v1, 0
+    v3 = load.i32 v2+8
+    v4 = heap_addr.i64 heap0, v1, 0
+    v5 = load.i32 v4+8
+    call fn0(v0)
+    v6 = load.i32 v4+8
+    v7 = load.i32 v4+8
+    return v3, v5, v6, v7
+}
```

Comment:
Could you add some comments about what cases this is testing and what is expected? That will be really helpful for folks updating tests for unrelated reasons and trying to determine whether the new precise output is valid or not. Also useful for reviewers trying to figure out which cases do or don't have tests ;)

Reply:
Done (here and in other new tests)!